linux_os/guide/system/network/network-nftables/set_nftables_base_chain/ansible/shared.yml

```diff
@@ -0,0 +1,71 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+#Name of the table
+{{{ ansible_instantiate_variables("var_nftables_table") }}}
+#Familiy of the table 
+{{{ ansible_instantiate_variables("var_nftables_family") }}}
+#Number of base chains
+{{{ ansible_instantiate_variables("var_nftables_base_chain_number") }}}
+#Name(s) of base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_names") }}}
+#Type(s) of base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_types") }}}
+# Hooks for base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_hooks") }}}
+#Priority
+{{{ ansible_instantiate_variables("var_nftables_base_chain_priorities") }}}
+#Policy 
+{{{ ansible_instantiate_variables("var_nftables_base_chain_policies") }}}
+
+- name: Remove file for remediation
```

Comment:
Please, include the rule title as prefix in the tasks names:
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible